oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticBulkProcessorHandler.java

```diff
@@ -268,113 +484,75 @@ public void beforeBulk(long executionId, BulkRequest request, List<String> conte
         }
 
         @Override
-        public void afterBulk(long executionId, BulkRequest request, List<String> contexts, BulkResponse response) {
+        public void afterBulk(long executionId, BulkRequest request, List<OperationContext> contexts, BulkResponse response) {
             try {
-                LOG.debug("Bulk with id {} processed in {} ms", executionId, response.took());
+                LOG.debug("Bulk with id {} processed in {} ms", executionId, response.took() / 1_000_000);
```

Comment:
`response.took()` returns milliseconds. Dividing it by 1M would result in `0` most of the times.

Reply:
Fixed, removed the division by 1000000. But `response.took()` seems to be returning bogus values, in my tests, I was always seeing the same arbitrary value. This value is received from the server, so most likely the problem is not on the client side.